Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM fedora:20
+FROM fedora:21
```

Comment:
Using implicit `:latest` is not best practice since it can lead to failures. I'd rather test on fedora:<next> then bump the Dockerfile `FROM` manually.